bin/populate-dependency-track

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+IFS='' read -d '' -r help_text << EOF
+$0
+
+Populates dependency track with SBOMs for the supplied repositories
+
+USAGE:
+$0 [FLAGS] <ARGUMENTS>...
+
+FLAGS:
+-t <target>: the name of the local fly target to use to submit jobs (**REQUIRED**)
+-c <resources-dir>: path to local repository checked out for ci-concourse-resources
+-d <dir>: path to local repository which contains checked out repos
+
+ARGUMENTS
+* Repos list file - a file containing a list of repos to generate SBOMs for
+  each line contains a different repository name.
+
+EOF
+
+function log() {
+    printf -- '[%s] - %s - %s\n' "$(date +'%FT%T%z')" "${1:?Level required}" "${2:?Message required}"
+}
+
+function clone_repo {
+    local repo_name="${1:?repo_name required}"
+    local branch_name="${2}"
+    log 'DEBUG' "cloning ${repo_name}"
+
+    if command -v gh > /dev/null 2>&1; then
+        if [[ -z "${branch_name}" ]]; then
+            gh repo clone companieshouse/"${repo_name}" "${local_dir}"/"${repo_name}"
+        else
+            gh repo clone companieshouse/"${repo_name}" "${local_dir}"/"${repo_name}" -- -b "${branch_name}"
+        fi
+    else
+        (
+            cd "${local_dir}" || exit 1
+
+            if [[ -z "${branch_name}" ]]; then
+                git clone git@github.com:companieshouse/"${repo_name}".git
+            else
+                git clone -b "${branch_name}" git@github.com:companieshouse/"${repo_name}".git
+            fi
+        )
+    fi
+}
+
+function resolve_version() {
+    local repository_name="$1"
+    (
+        cd "${local_dir}"/"${repository_name}" || exit 1
+
+        gh release view --json name --jq .name || git describe --tags --abbrev=0 || cat version
+    )
+}
+
+function set_parameters() {
+    DEP_TRACK_SERVER='((secrets.dependency-track-server))'
+    DEP_TRACK_API_KEY='((secrets.dependency-track-api-key))'
+    OTHER_CDXGEN_ENV_VARS=""
+    DEP_TRACK_PROJ_TAGS="auto-populated;upload-$(date +'%Y-%m-%dT%H:%M:%S')"
+    SEND_TO_DEP_TRACK="true"
+    OTHER_CDXGEN_ENV_VARS='ANOTHER_ENV_VAR=yes'
```

Comment:
If you set it here then you can remove the one 3 lines above (line 62)
I think if you remove the setting of this var altogether (both lines) it might work as well